projects/observability/src/shared/dashboard/widgets/gauge/gauge-widget.model.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Color } from '@hypertrace/common';
+import { createModelFactory } from '@hypertrace/dashboards/testing';
+import { MODEL_PROPERTY_TYPES } from '@hypertrace/hyperdash-angular';
+import { runFakeRxjs } from '@hypertrace/test-utils';
+import { of } from 'rxjs';
+import { GaugeWidgetData } from './gauge-widget';
+import { GaugeWidgetModel } from './gauge-widget.model';
+
+describe('Gauge widget model', () => {
+  test('uses colors from color map', () => {
+    const modelFactory = createModelFactory();
+
+    const data: GaugeWidgetData = {
+      value: 5,
```

Comment:
Hmm - I think I'd expect the max and thresholds to come from the widget model rather than the data model - those seem like they're independent of where we actually get the value from, and would allow dropping in our existing aggregation data sources.

Reply:
That's a valid point. The reason I did it this way is because the api - risk graphql exposes a risk value and a risk category. So the data source I built, would convert the risk category into one of these thresholds. But I agree, these two should come from widget model. Let me talk to Ines and establish different threshold values for the risk.